### PR TITLE
Data tests

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Improved coverage of the data package to 97% (line)

### CODE REVIEW

File: src/main/java/data/ScriptingProject.java [hunk 1]
```diff
@@ -249,7 +244,7 @@ public double secondsToCounts(double seconds) {
      */
     public Set<CameraType> getDistinctCameraTypes() {
         Set<CameraType> result = new HashSet<CameraType>();
-        for (Camera c: cameras) {
+        for (Camera c: this.cameras) {
```
Comment (mennoo1996) [inline src/main/java/data/ScriptingProject.java:247]: Why did you add this?

Comment (bart-de-jonge) [inline src/main/java/data/ScriptingProject.java:247]: Readability, I caught myself looking for where cameras was defined

**Comment by mennoo1996** [comment]
Nice work!

**Comment by Markvanderuit** [comment]
Looks good, merging...
